### DISCUSSION TITLE
[질문] APP 모듈에 있는 gradle 파일 오류

### DISCUSSION THREAD

**Discussion by boogi-woogi** [TEAM-DAME/Dame-Android#109] [2. Q&A]
![image](https://user-images.githubusercontent.com/88770426/204096057-93dd9b63-7402-472e-8e4a-1ec3bb1f714c.png)
갑자기 이 부분에서 오류가 발생했습니다. 따로 수정하거나 전혀 건드린 부분이 없는데 해결방법을 잘 모르겠습니다 ㅠㅠ.
Clean project와 rebuild 모두 시도해보았습니다!

![image](https://user-images.githubusercontent.com/88770426/204096588-18ba523d-37ac-4ce3-b103-c3f09f4e0b34.png)

<일단 해당 부분을 주석처리하고 진행하겠습니다!>

**Comment by onseok** [comment]
`local.properties` 파일에 네이버 클로바 감정분석 API 연동을 위한  `X_NAVER_CLIENT_ID`, `X_NAVER_CLIENT_SECRET` 키 값이 빠져있는지 확인부탁드립니다.
